src/dataset.py

```diff
@@ -95,6 +96,13 @@ def normalize_and_mask(cls, eo: np.ndarray):
         normed_eo = np.where(eo[:, keep_indices] != cls._NODATAVALUE, normed_eo, 0)
         return normed_eo
 
+    @staticmethod
+    def test_val_split(df: pd.DataFrame, seed=DEFAULT_SEED) -> Tuple[pd.DataFrame, pd.DataFrame]:
```

Comment:
As discussed on slack, do we need this?

Reply:
I thought having the function wouldn't hurt (it's not used) but I'll remove it!